.github/workflows/lint.yml

```diff
@@ -0,0 +1,49 @@
+name: Lint
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  run-linters:
+    name: Run Linters
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Cache yarn dependencies
```

Comment:
Does it work on GitHub action to store a cache? Or is this just something you run locally?